discojs/src/aggregator/byzantine.ts

```diff
@@ -0,0 +1,185 @@
+import createDebug from "debug";
+import { List, Map } from "immutable";
+
+import { AggregationStep, Aggregator } from "./aggregator.js";
+import { WeightsContainer, client } from "../index.js";
+import { aggregation } from "../index.js";
+import * as tf from '@tensorflow/tfjs'
+
+
+const debug = createDebug("discojs:aggregator:mean");
+
+type ThresholdType = 'relative' | 'absolute'
+
+/** 
+ * Mean aggregator whose aggregation step consists in computing the mean of the received weights. 
+ * 
+ */
+export class ByzantineRobustAggregator extends Aggregator {
```

Comment:
can you add some information on what it is based? (paper URL, assumption required to use it safely, tradeoffs, …)